src/lib.rs

```diff
@@ -45,3 +45,38 @@ impl fmt::Display for NonSortedIntegers {
 }
 
 impl Error for NonSortedIntegers {}
+
+/// An [`IntoIterator`] blanket implementation that provides extra methods for [`RoaringBitmap`]
+/// and [`RoaringTreemap`]
+///
+/// This trait is parameterized by a sealed trait and cannot be implemented for types outside
+/// of this crate
+pub trait IterExt<T>: IntoIterator<Item = T>
+where
+    T: private::Roaring,
```

Comment:
I am not sure why you want to restrict this trait to our library types? What's dangerous in implementing that on, for example, a wrapper of `RoaringBitmap/Treemap`?

Reply:
Adding new non-defaulted methods would be a breaking change downstream. I was thinking this was for public API compatibility.